.github/workflows/main.yaml

```diff
@@ -0,0 +1,23 @@
+name: release workflow
+on:
+  push:
+    tags:
+      - 'v*'
+jobs:
+  build:
+    name: release action
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@master
+    - uses: actions/setup-go@v1
+      with:
+        go-version: '1.12'
+    - name: zip windows executable
```

Comment:
the name looks a bit stale given other changes

Reply:
Fixed.